src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -185,8 +185,7 @@ define(function (require, exports, module) {
             callback = options;
         }
         options = options || {};
-
-        var encoding = options.encoding || "utf8";
+        options.encoding = options.encoding === null ? null : "utf8";
```

Comment:
Correct.  Within the context of Brackets, utf8 is by far the most common case, and is assumed throughout much of the Adobe code.